### ISSUE TITLE
UnicodeDecodeError when installing on python3 via pip3

### ISSUE THREAD

**Issue by mborho** [fritzy/SleekXMPP#286]
Hi,

when installing SleekXMPP with pip3 on python3.3, I get the following traceback:

```
root@8733a2aaeb62:/# pip3 install sleekxmpp
Downloading/unpacking sleekxmpp
  Downloading sleekxmpp-1.2.4.tar.gz (837kB): 837kB downloaded
  Running setup.py egg_info for package sleekxmpp

Cleaning up...
Exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 134, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 236, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/usr/lib/python3/dist-packages/pip/req.py", line 1139, in prepare_files
    req_to_install.assert_source_matches_version()
  File "/usr/lib/python3/dist-packages/pip/req.py", line 394, in assert_source_matches_version
    version = self.installed_version
  File "/usr/lib/python3/dist-packages/pip/req.py", line 390, in installed_version
    return self.pkg_info()['version']
  File "/usr/lib/python3/dist-packages/pip/req.py", line 357, in pkg_info
    data = self.egg_info_data('PKG-INFO')
  File "/usr/lib/python3/dist-packages/pip/req.py", line 297, in egg_info_data
    data = fp.read()
  File "/usr/lib/python3.3/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 540: ordinal not in range(128)
```

Cheers
Martin

**Comment by lordi** [comment]
Setting your shell's $LANG to a UTF-8 encoding fixes this. Having said that, this needs to be fixed for all encodings.

I suspect this is due to the inclusion of readme.rst in 
https://github.com/fritzy/SleekXMPP/blob/develop/setup.py#L35

**Comment by bear** [comment]
This was fixed in another PR - tested with Python v3.4 and the latest PyPI --

milo~/temp: $ virtualenv-3.4 foobar
Using base prefix '/Users/bear/homebrew/Cellar/python3/3.4.3/Frameworks/Python.framework/Versions/3.4'
New python executable in foobar/bin/python3.4
Also creating executable in foobar/bin/python
Installing setuptools, pip...done.
milo~/temp: $ cd foobar/
milo~/temp/foobar: $ . bin/activate
(foobar)milo~/temp/foobar: $ bin/pip3 install sleekxmpp
Collecting sleekxmpp
  Downloading sleekxmpp-1.3.1.tar.gz (831kB)
    100% |████████████████████████████████| 831kB 489kB/s
Installing collected packages: sleekxmpp
  Running setup.py install for sleekxmpp
Successfully installed sleekxmpp-1.3.1
